src/pages/Homepage/HeroSlideshow.vue

```diff
@@ -82,6 +107,7 @@ const mgPromoImageRequire = require.context('@/assets/images/mg-hero-slideshow/'
 
 export default {
 	name: 'HeroSlideshow',
+	// I need to return the props.doubleArrowButtonExp like the following example is doing for mgPromoExp
```

Comment:
For this you can add some logic above the return statement to determine which of your 4 cases should be the cacheKey. Make sure to give a unique key for your 2 new cases...then just return you final key.

Reply:
👍